docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-113.adoc

```diff
@@ -0,0 +1,47 @@
+== MonkeyLearn API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_113
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A MonkeyLearn API Key is essential for accessing the MonkeyLearn text analysis services, which provide powerful machine learning models for text data. The key is critical for authenticating requests to the API and ensuring that only authorized applications can utilize the service. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. It is vital to manage this key securely and regularly review your security practices to prevent any unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://monkeylearn.com/api/v3/#authentication[MonkeyLearn documentation and support] resources.
+
+=== Fix - Buildtime
+
+*MonkeyLearn*
+
+If your MonkeyLearn API Key is exposed, it's important to act promptly to revoke the compromised key and generate a new one to secure access to the API. Follow these steps to replace the exposed key and update all dependent applications:
+
+1. Log into your MonkeyLearn account using your administrator credentials.
+
+2. Navigate to the 'API Keys' section within your account settings.
+
+3. Identify the exposed key and delete it.
+
+4. Generate a new API key by selecting 'Create new key'. Provide a name for the new key and save the changes.
```

Comment:
Select 'Create new key' to generate a new API Key. Provide a name for the new key and save the changes.